maquetado/scss/detail.scss

```diff
@@ -0,0 +1,58 @@
+%equal-font-heights {
+ 
+  font-family: Roboto;
```

Comment:
When you use a font-family, always remember to set a "back up" family.

`font-family: Roboto, sans-serif;`

That way if the font is not properly loaded, the browser will use that back-up 😄